galsim/lensing_ps.py

```diff
@@ -423,6 +423,14 @@ def buildGrid(self, grid_spacing=None, ngrid=None, rng=None, interpolant=None,
                                 does not modify the internally-stored power spectrum, just the
                                 shears generated for this particular call to buildGrid().
                                 [default: "hard"]
+        @param variance         Optionally renormalize the variance of the output shears to a
+                                given value.  This is useful if you know the functional form of
+                                the power spectrum you want, but not the normalization.  This lets
+                                you set the normalization separately.  The resulting shears should
+                                have var(g1) + var(g2) ~= variance.  If only e_power_function is
+                                given, then this is also the variance of kappa.  Otherwise, the
+                                variance of kappa may be smaller than the specified variance.
```

Comment:
Hmm.  That seems rather subjective.  How small is too small?  

Also, I think the variance that results from this procedure will be correct by construction.  Just the measured variance may not match the expectation of the variance for that power spectrum input.  But it will measure whatever variance actually results from the gaussian field and scale the signal accordingly to give it the specified variance.  

So I guess I'm not even sure what phrasing you would want for the warning.  I'm pretty sure the parameter does work exactly as advertised.